sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -578,6 +578,21 @@ namedExpressionSeq
     : namedExpression (',' namedExpression)*
     ;
 
+transformList
+    : '(' transforms+=transform (',' transforms+=transform)* ')'
+    ;
+
+transform
+    : qualifiedName                                                           #identityTransform
```

Comment:
it's used in CREATE TABLE statement only, do we really need `qualifiedName`? I think `identifier` is good enough here.

Reply:
I think it is better to use `qualifiedName`. This may be a logical name in the current use, but later Spark may need to resolve the transform using this name. For example, this could be set to `builtin.bucket` to tell Spark that it is the built-in bucket transform function. Using that information, Spark would know it can run a bucketed join.

Reply:
How about the transform arguments? Do they need to be `qualifiedName` as well?

Reply:
Yes, arguments need to be `qualifiedName` because they may reference nested fields.